proto/greptime/v1/ddl.proto

```diff
@@ -90,6 +90,16 @@ message RenameTable { string new_table_name = 1; }
 message AddColumn {
   ColumnDef column_def = 1;
   bool is_key = 2;
+
+  message Location {
+    enum LocationType {
+      FIRST = 0;
+      AFTER = 1;
+    }
+    LocationType location_type = 1;
+    string after_cloumn_name = 2;
+  }
+  Location location = 3;
```

Comment:
question(non-blocking): Is it possible to use oneof? Which style is more clear?

Reply:
I am not quite sure either. Used to using enum. Maybe we can wait for more conversation about this.

Reply:
I prefer `enum`.